### PR TITLE
import pre setup command base code

### PR DESCRIPTION
**Pull request by shafeeqd959**

### CODE REVIEW

File: packages/contentstack-import-setup/messages/index.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (sunil-lakshman) [inline packages/contentstack-import-setup/messages/index.json:1]: @shafeeqd959 what purpose we used this empty object file.

Comment (shafeeqd959) [reply packages/contentstack-import-setup/messages/index.json:1]: Will add messages in future commits

File: packages/contentstack-import-setup/src/utils/common-helper.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+      let entity = e;
+      if (e === 'authorization') entity = 'Management Token';
+      if (e === 'api_key') entity = 'Stack API key';
+      if (e === 'uid') entity = 'Content Type';
+      if (e === 'access_token') entity = 'Delivery Token';
+      message += ' ' + [entity, error.errors[e]].join(' ');
+    });
+  }
+  return message;
+};
```
Comment (aman19K) [inline packages/contentstack-import-setup/src/utils/common-helper.ts:21]: @shafeeqd959 , We can utilise formatError, which we created in utilities.

**Review by aman19K — APPROVED** [review]

File: packages/contentstack-import-setup/package.json [hunk 3]
```diff
@@ -0,0 +1,97 @@
+{
+  "name": "@contentstack/cli-cm-import-setup",
+  "description": "Contentstack CLI plugin to setup the mappers and configurations for the import command",
+  "version": "1.0.0",
+  "author": "Contentstack",
+  "bugs": "https://github.com/contentstack/cli/issues",
+  "dependencies": {
+    "@contentstack/cli-command": "~1.3.0",
+    "@contentstack/cli-utilities": "~1.7.1",
+    "@contentstack/management": "~1.17.0",
```
[the remark this answers is not in the document]
Comment (shafeeqd959) [reply packages/contentstack-import-setup/package.json:10]: Will remove in the upcoming commits

**Pull request merged**